theme.html

```diff
@@ -0,0 +1,284 @@
+            class="fab fa-linkedin"></i></a>
+   </div>
+   <!-- Section for social media links ends -->
+  <div class="box">
+   <div class="box-inner px-1">
+      This website will be offline for maintenance and updates from 01:30 AM to 02:00 AM IST.
+   </div>
+   </div>
+   <button onclick="topFunction()" id="myBtn" title="Go to top"><i class="fa fa-angle-up"
+         aria-hidden="true"></i></button>
+  <nav class="navbar navbar-expand-lg site-header sticky-top py-1">
+      <a class="navbar-brand" href="#"><span class="navbar-toggler-icon"></span></a>
+      <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent"
+         aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+         <span class="navbar-toggler-icon" style="color:#f50000"><i class="fas fa-align-justify"></i></span>
+      </button>
+      <div class="collapse navbar-collapse" id="navbarSupportedContent">
+         <ul class="navbar-nav mr-auto  mx-auto navbar-right">
+            <li class="nav-item active">
+               <a class="nav-link" href="#">Home <span class="sr-only">(current)</span></a>
+            </li>
+            <li class="nav-item">
+               <a class="nav-link" href="#about">About</a>
+            </li>
+           <!--  <li class="nav-item">
+               <a class="nav-link" href="#register">Registration</a>
+            </li> -->
+            <!-- <li class="nav-item">
+               <a class="nav-link" href="resources.html">Resources</a>
+            </li> -->
+            <!-- <li class="nav-item">
+               <a class="nav-link" href="submissions.html">Submissions</a>
+            </li> -->
+            <!-- <li class="nav-item">
+               <a href="results.html" class="nav-link">Results</a>
+            </li> -->
+             <!-- <li class="nav-item">
+               <a class="nav-link" href="#">Schedule</a>
+            </li> -->
+            <li class="nav-item">
+               <a class="nav-link" href="#endorsement-partners">Endorsement Partners</a>
+            </li>
+            <li class="nav-item">
+               <a class="nav-link" href="#partners-and-organisers">Partners and Organisers</a>
+            </li>
+            <!-- <li class="nav-item">
+               <a class="nav-link" href="#rules-and-regulations">Rules and Regulations</a>
+            </li> -->
+             <li class="nav-item">
+               <a class="nav-link" href="#">Rewards</a>
+            </li>
+            <li class="nav-item">
+               <a class="nav-link" href="#faqs">FAQs</a>
+            </li>
+            <li class="nav-item">
+               <a class="nav-link" href="#contact-us">Contact us</a>
+            </li>
+         </ul>
+      </div>
+   </nav> 
+   <br>
+ <h2><center><p style="color: purple" id="demo"></p></center></h2>
+
+<script>
```

Comment:
Add the script in custom.js file and please check if it is already present

Reply:
Done